### DISCUSSION TITLE
How do I transfer the texture of one pipeline to another (for v7)?

### DISCUSSION THREAD

**Discussion by AndreyPrakhov** [pmndrs/postprocessing#623] [Q&A]
What's the problem... Deferred shading is performed, the result of which must then be smoothed. SMAA or FXAA shaders work directly with GBuffer.color 

Most likely, I need to create two pipelines. One will create a GBuffer and perform deferred shading and fill in the texture. Then the second pipeline already applies anti-aliasing.

The question arises how to correctly transfer the result of the first pipeline in the form of GBuffer.color for the second (it is clear that it would be possible to modify the code a little, but I would not like to)

**Comment by vanruesc** [comment]
`GBuffer.color` is the input color from the previous pass. Don't overcomplicate things. You can simply chain two EffectPasses: one that does the lighting and one that applies effects and AA.
